entry.sh

```diff
@@ -2,4 +2,4 @@
 
 echo '{"irods_host": "data.cyverse.org", "irods_port": 1247, "irods_zone_name": "iplant"}' | tee  > /home/user/.irods/irods_environment.json
```

Comment:
One small thing I didn't do but could be done: in VICE, there should be an environment variable IPLANT_USER that has the job user's username in it, which could be put into the `irods_user_name` configuration here. Then, when they run iinit or whatever, it'll just ask for password, not username.